src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```
@@ -33,10 +33,10 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <!-- Paths to tools, tasks, and extensions are calculated relative to the BlazorWebAssemblySdkDirectoryRoot. This can be modified to test a local build. -->
     <BlazorWebAssemblySdkDirectoryRoot Condition="'$(BlazorWebAssemblySdkDirectoryRoot)'==''">$(MSBuildThisFileDirectory)..\</BlazorWebAssemblySdkDirectoryRoot>
-    <_BlazorWebAssemblySdkTasksTFM Condition=" '$(MSBuildRuntimeType)' == 'Core'">net5.0</_BlazorWebAssemblySdkTasksTFM>
+    <_BlazorWebAssemblySdkTasksTFM Condition=" '$(MSBuildRuntimeType)' == 'Core'">$(DefaultNetCoreTargetFramework)</_BlazorWebAssemblySdkTasksTFM>
```

While this file may now work when used in our tests (due to https://github.com/dotnet/aspnetcore/pull/26788/files#diff-513b95dc6e85629e2a29eafc523e38cad380806100484a3901bf16058071bceaR15), this change is incorrect in this (shipping) file. Suggest
```suggestion
    <_BlazorWebAssemblySdkTasksTFM Condition=" '$(MSBuildRuntimeType)' == 'Core'">net6.0</_BlazorWebAssemblySdkTasksTFM>
```

The general problem is similar to problems that crop up when building things in folders under Directory.Build.* files not everyone is aware of. Need to look around when stepping into these dusty corners.

---

yes please. These files are part of the SDK and cannot use AspNetCore's build variables.